src/System.Management.Automation/engine/CommandDiscovery.cs

```diff
@@ -258,6 +258,7 @@ internal void AddSessionStateCmdletEntryToCache(SessionStateCmdletEntry entry, b
         /// </param>
         /// <param name="commandOrigin">Location where the command was dispatched from.</param>
         /// <param name="useLocalScope">
+        /// <param name="forCompletion">
```

Comment:
You insert new block to existing block. Please move below, add description and add closing part.